xmake/toolchains/ndk/load.lua

```diff
@@ -166,6 +166,8 @@ function main(toolchain)
         -- add sysroot flags
         local ndk_sysroot = toolchain:config("ndk_sysroot")
         if ndk_sysroot and os.isdir(ndk_sysroot) then
+            -- use unix path separator on android as windows seps may not be understood by some tools (e.g. openssl's perl)
+            ndk_sysroot = path.unix(ndk_sysroot)
```

Comment:
I don't think it's a good idea to globally change the toolchain for a specific package build file. it's a bit of a hack.

Reply:
we should modify tools.autoconf and tools.cmake to support it instead of it.

Reply:
done